### PR TITLE
V0 clean up

### PR DESCRIPTION
**Pull request by fredrikjacobson**
Several unused methods removed to facilitate use of input API

Annoutil has also been simplified to include fewer commands. Now has four commands:
- projects
- inputs
- viewlink
- calibration

also added backporting of dataclasses so that it should work for python 3.6.
I have yet to confirm this..

### CODE REVIEW

File: annotell-input-api/annotell/input_api/input_api_model.py [hunk 1]
```diff
@@ -211,7 +213,7 @@ def __init__(self, external_id: str,
                  sequence_id: int,
                  sub_sequence_id: int,
                  settings: Optional[dict] = None):
-        super().__init__(external_id, source_specification, calibration_id)
+        super().__init__(external_id, calibration_id, source_specification)
         self.vehicle_data = vehicle_data
```
Comment (Ragge-dev) [inline annotell-input-api/annotell/input_api/input_api_model.py:217]: Vem det än var, good catch!

File: annotell-input-api/annotell/input_api/input_api_model.py [hunk 2]
```diff
@@ -497,7 +496,7 @@ def __init__(self, id, external_id: str, source: Optional[str], created: datetim
     def from_json(js: dict):
         return Data(
             int(js["id"]),
-            js.get("externalId"),
+            str(js.get("externalId")),
             js.get("source"),
             ts_to_dt(js["created"])
         )
```
Comment (Ragge-dev) [inline annotell-input-api/annotell/input_api/input_api_model.py:502]: Vi kan väl fimpa hela denna?

**Review by Ragge-dev — COMMENTED** [review]
Tror vi kan plocka bort hela InputJob modellen? Annars skeppbart säger jag!

File: annotell-input-api/annotell/input_api/input_api_model.py (hunk 2 above)
Comment (Ragge-dev) [reply annotell-input-api/annotell/input_api/input_api_model.py:502]: Fick inte med allt, men detta är alltså en del av InputJob modellen

Comment (fredrikjacobson) [reply annotell-input-api/annotell/input_api/input_api_model.py:502]: hmm, sure why not. 
lets do it

**Pull request merged**